draft-high-assurance-profile-oid4vc-sd-jwt-vc.md

```diff
@@ -54,10 +56,11 @@ ToDo: Mostly reference other specs.
 # Scope
 
 The following aspects are in scope of this interoperability profile:
-* Protocol for issuance of the Credentials online (can be both remote and in-person)
-* Protocol for presentation of Credentials online (can be both remote and in-person)
-* User Authentication
-* Wallet Instance Attestation (during Credential issuance)
+
+* Protocol for issuance of the Credentials online (can be both remote and in-person) (OID4VCI)
+* Protocol for presentation of Credentials online (can be both remote and in-person) (OID4VP)
+* Protocol for User Authentication with the Wallet at a Verifier (SIOP v2)
+* Wallet Attestation (during Credential issuance)
```

Comment:
because I think the aim is to attest the wallet, not a certain instance - An issuer needs to know it is talking to a trustworthy wallet (wallet provider/product), not a certain instance

Reply:
ok..